services/dy-static-file-server/docker/custom/Dockerfile

```diff
@@ -30,16 +38,33 @@ RUN python3 -m venv /venv
 COPY requirements/base.txt /tmp/requirements.txt
 RUN /venv/bin/pip3 install -r /tmp/requirements.txt
 
-COPY static-content/hello-world.txt /www/hello-world.txt
 
 RUN mkdir -p ${INPUT_DIR} && \
     mkdir -p ${OUTPUT_DIR} && \
     mkdir -p ${WORKDIR}
 
+COPY static-content/hello-world.txt /www/hello-world.txt
 COPY src/dy_static_file_server ${WORKDIR}/dy_static_file_server
 
+## add permissions
+RUN chown -R ${USER}:${USER} /boot.sh && \
+    chown -R ${USER}:${USER} /var/cache/nginx && \
+    chown -R ${USER}:${USER} /var/log/nginx && \
+    chown -R ${USER}:${USER} /etc/nginx/conf.d && \
+    chown -R ${USER}:${USER} /etc/nginx/conf.d/default.conf && \
+    chown -R ${USER}:${USER} ${INPUT_DIR} && \
+    chown -R ${USER}:${USER} ${OUTPUT_DIR} && \
+    chown -R ${USER}:${USER} ${WORKDIR}
```

Comment:
same as above. I would use the entrypoint.sh for this, which is as we found out, what the computational services are doing.

Reply:
no longer applies